lua/plenary/async/async.lua

```diff
@@ -6,6 +6,10 @@ local f = require "plenary.functional"
 
 local M = {}
 
+local function is_callable(fn)
+  return type(fn) == "function" or getmetatable(fn)["__call"]
```

Comment:
the part after or is not a comparison. It will return the value behind `__call` key or nil. Should probably also be checked against type

Reply:
Done.